### PR TITLE
Update Getting Started

### PR DESCRIPTION
**Pull request by allc**
Fix indentation so the lists should show correctly on Read The Docs

  - The style guides say to have two spaces for indentation, I used four spaces here just to be consistent with many of other pages in this guide (at least also with rest of this page)

Mention command for generating configurations in VS Code
Mention to edit `eula.txt` for testing the server

  - Think it is helpful to mention it, and it does not seem to appear in the error message

### CODE REVIEW

**Review by Aeltumn — DISMISSED** [review]
Good change to also mention you can use VS Code now. Although I don't know if it's important to mention you have to accept the eula, that should be obvious.

**Comment by tterrag1098** [comment]
The indentation was fixed by #301, move this to 1.14.x and apply only your content changes, and this would still be okay to merge.

**Comment by allc** [comment]
@tterrag1098 Thanks for the comment.
I have changed the base branch to 1.14.x and edited the commits so there is only content changes in this PR.

File: docs/gettingstarted/index.md [hunk 1]
```diff
@@ -14,12 +14,13 @@ From Zero to Modding
     * the `gradle` folder
 3. Move the files listed above to a new folder, this will be your mod project folder.
 4. Choose your IDE:
-    * Forge explicitly supports developing with Eclipse or IntelliJ environments, but any environment, from Netbeans to vi/emacs, can be made to work.
+    * Forge explicitly supports developing with Eclipse, IntelliJ or Visual Studio Code environments, but any environment, from Netbeans to vi/emacs, can be made to work.
```
Comment (sciwhiz12) [inline docs/gettingstarted/index.md:17]: Forge does not officially provide support for developing in Visual Studio Code. Although the run task exists, it is expected that users who wish to use VS Code for making Forge mods must solve any issues that arise from their use of VS Code by themselves (or through other community members).

Please revert this change, and add an admonition after the `genVSCodeRuns` paragraph to warn users of this.
(See the `Customizing Your Mod Information` section for an example of an admonition, the `!!! important` part)

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]
Please rebase this PR to the 1.15.x branch, as 1.14.x is out-of-support.
_(If you do not wish to continue this PR, please respond soon.)_

**Comment by ChampionAsh5357** [comment]
Merged in the omnibus #364.

**Pull request closed without merging**